pkg/clients/db_client.go

```diff
@@ -15,9 +15,11 @@ type DBClient struct {
 func NewDBClient(host, user, password string, port int, database, application_name_suffix, version, sslmode string) (*DBClient, diag.Diagnostics) {
 	var diags diag.Diagnostics
 
-	application_name := fmt.Sprintf("terraform-provider-materialize v%s", version)
+	var application_name string
 	if application_name_suffix != "" {
-		application_name += " " + application_name_suffix
+		application_name = application_name_suffix
```

Comment:
It seems confusing to call this _suffix if it's actually the full application name?

Reply:
Good point! I changed this a bit as it indeed did not make sense to have it this way